app/scripts/proactive/view/CatalogPublishView.js

```diff
@@ -379,7 +389,7 @@ define(
 
             if( typeof alreadyPublishedBucketName === "undefined"){
                 var studioApp = require('StudioApp');
-                const bucketNameObject = studioApp.models.jobModel.get("Generic Info").filter(function(item){
+                const bucketNameObject = studioApp.models.jobModel.attributes["Generic Info"].filter(function(item){
```

Comment:
why did you have to change this ?

Reply:
Because it throwed an unknown function error. I will test again.

Reply:
Ok it works also without this update.